lib/src/facts/windows/operating_system_resolver.cc

```diff
@@ -100,8 +100,8 @@ namespace facter { namespace facts { namespace windows {
         // Override default release with Windows release names
         auto version = result.release.substr(0, lastDot);
         bool consumerrel = (wmi::get(vals, wmi::producttype) == "1");
-        if (version == "6.4") {
```

Comment:
So one question that comes to mind here.... What about Windows 10 anniversary update?  That build has different behavior than the existing Windows 10 builds (and appears to start around build 14295).

Microsoft's intent is that that it should be no different, but the reality is that it does have some different behavior (at least with respect to PowerShell).

I'm not sure that we have a good solution ... maybe we just punt on worrying about that?

Any thoughts @puppetlabs/windows ?